unit-tests/debug_protocol/test-build-raw-data.py

```diff
@@ -0,0 +1,53 @@
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+
+def convert_bytes_to_decimal(command):
+    command_input = []  # array of uint_8t
+
+    # Parsing the command to array of unsigned integers(size should be < 8bits)
+    # threw out spaces
+    command = command.lower()
+    command = command.split()
+
+    for byte in command:
+        command_input.append(int('0x' + byte, 0))
+
+    return command_input
+
+
+def send_hardware_monitor_command(device, command):
+    raw_result = rs.debug_protocol(device).send_and_receive_raw_data(command)
+    return raw_result[4:]
+
+
+test.start("Init")

```

Comment:
Not a good test name...
It's OK to have an `Init` -- you can put the `ctx` and `dev` initialization there if you want...
You can even separate the two scenarios into different test-cases.